samples/transcribeContextClasses.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+function main(storageUri) {
+  // [START speech_transcribe_sync]
+  // Provides "hints" to the speech recognizer to favor
+  // specific classes of words in the results.
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech');
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  async function transcribeContextClasses() {
+    // storageUri = 'gs://YOUR_BUCKET_ID/path/to/your/file.wav'
+    const audio = {
+      uri: storageUri,
+    };
+
+    // SpeechContext: to configure your speech_context see:
+    // https://cloud.google.com/speech-to-text/docs/reference/rpc/google.cloud.speech.v1#speechcontext
+    // Full list of supported phrases(class tokens) here:
+    // https://cloud.google.com/speech-to-text/docs/class-tokens
+    const speechContext = {
+      phrases: ['$TIME'],
+    };
+
+    // RecognitionConfig: to configure your encoding and sample_rate_hertz, see:
+    // https://cloud.google.com/speech-to-text/docs/reference/rpc/google.cloud.speech.v1#audioencoding
```

Comment:
Mea culpa I had the link wrong in the canonical (since fixed).

Should be https://cloud.google.com/speech-to-text/docs/reference/rpc/google.cloud.speech.v1#recognitionconfig

Reply:
Fixed